Common/AssemblyResolver.cs

```diff
@@ -59,7 +69,13 @@ public AssemblyResolver(ILogger logger, params string[] rootSearchPaths)
                 {
                     asm = Assembly.LoadFile(file);
 
-                    if (string.Equals(args.Name, asm.FullName, StringComparison.CurrentCultureIgnoreCase))
```

Comment:
Should using OrdinalIgnoreCase when comparing file names: https://msdn.microsoft.com/en-us/library/dd465121(v=vs.110).aspx